st2client/st2client/formatters/table.py

```diff
@@ -248,3 +249,13 @@ def _get_attribute_value(subject, attribute):
         if isinstance(r_val, list) or isinstance(r_val, dict):
             return r_val if len(r_val) > 0 else ''
         return r_val
+
+
+class SingleRowTable(object):
+
+    @staticmethod
+    def note_box(message):
+        note = PrettyTable([""])
+        note.header = False
+        note.add_row([message])
+        return sys.stderr.write(str(note))
```

Comment:
Missing line break at the end.

It would also be good if you could align note box table with actual content table (so it's the same number of columns / width).

![screenshot from 2017-06-29 11-43-54](https://user-images.githubusercontent.com/125088/27681657-4816d99e-5cc0-11e7-897a-6a1a6b286abc.png)